jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/RequestLogger.java

```diff
@@ -61,24 +69,33 @@ public static void headersToString(HttpServerResponse response, StringBuilder lo
      * @param header The header to get the value from
      * @return value of the header or "REDACTED" if the header contains sensitive information
      */
-    public static String getHeaderValueOrRedact(Map.Entry<String, String> header) {
-        return getHeaderValueOrRedact(header, emptySet());
+    public String getHeaderValueOrRedactIncoming(Map.Entry<String, String> header) {
```

Comment:
Incoming/Outgoing might be problematic because both client and server has both a request and a response with headers. Is this intentional (outgoing being response of server and request of client) or not? Otherwise I suggest using "server"/"client".

Reply:
server/client is better.